big-red-shuttle/StopsViewController.swift

```diff
@@ -195,6 +193,12 @@ class StopsViewController: UIViewController, UITableViewDelegate, UITableViewDat
     
     
     func mapView(_ mapView: GMSMapView, didTapAt coordinate: CLLocationCoordinate2D) {
+        for marker in markers {
+            let iconView = marker.iconView as! IconView
+            if iconView.clicked! {
+                markerUnselectedAnimation(iconView: iconView)
+            }
+        }
```

Comment:
Instead of having an array of `markers` and looping through ever `marker`for each tap, there should be just one `selectedMarker`